### ISSUE TITLE
use-vue3-easy-data-table does not come with types declaration for typescript

### ISSUE THREAD

**Issue by therealokoro** [HC200ok/vue3-easy-data-table#85]
Hello, so i'm trying to modify customize the footer of the table using the `use-vue3-easy-data-table` package but once installed, vscode typescript is shouting at me because it doesn't come with types which is sad...😔

In your docs you import some type definition from the package, but i couldn't find that type def. Please look into this. Thank u 🙂
Screenshot below 👇👇

![dddd](https://user-images.githubusercontent.com/41494675/181342070-25202531-ef6b-4a94-94a6-1928d7b5c1bb.png)

**Comment by HC200ok** [comment]
@therealokoro Thank you for reminding, please update to the latest version `0.1.2` and try again.

**Comment by therealokoro** [comment]
> @therealokoro Thank you for reminding, please update to the latest version `0.1.2` and try again.

I will, thank you for the fix 😊.

Can you consider making this package part of the main stuff, so we don't have to install two packages

**Comment by HC200ok** [comment]
> Can you consider making this package part of the main stuff, so we don't have to install two packages

Emm... Actually, I designed `vue3-easy-data-table` as a pure component library, so after installing `vue3-easy-data-table`, then you import globally or in an SFC and use it
```vue
import Vue3EasyDataTable from 'vue3-easy-data-table';
```

`use-vue3-easy-data-table` is a library that contains some composable functions to customize `vue3-easy-data-table`. the size of `use-vue3-easy-data-table` may become bigger, but it is not a must for many users. So I separated it from the main stuff. But one package is more simple and clean, indeed. I will keep comparing these two patterns and notice you if I make them as one.
